packages/eslint-plugin/configs/recommended-with-formatting.js

```diff
@@ -20,14 +20,23 @@ const config = {
 		document: true,
 		wp: 'readonly',
 	},
+	settings: {
+		'import/internal-regex': '^@wordpress/',
+		'import/extensions': [ '.js', '.jsx' ],
+	},
 	rules: {
 		'import/no-extraneous-dependencies': [
 			'error',
 			{
 				peerDependencies: true,
 			},
 		],
-		'import/no-unresolved': 'error',
+		'import/no-unresolved': [
+			'error',
+			{
+				ignore: [ '^@wordpress/' ],
```

Comment:
Is there a way in which we can exclude packages like the '@wordpress/icons' package from this ignore lists? It already is confusing which ones are included in core and which ones aren't.

Reply:
Good catch. It's also `@wordpress/interface`. I completely forgot about them. I'm positive we can tweak the regular expression 👍🏻

Reply:
https://github.com/WordPress/gutenberg/pull/37639/commits/2360498eebb4e1afc1b47e4634d8b0e4c856241f should do the trick.